src/app/model/User.ts

```diff
@@ -0,0 +1,107 @@
+import { UserInterface, VolunteerStatus, Location, MissionStatus } from "./schema";
+import BaseModel from "./BaseModel";
+
+const defaultLocation: Location = {
```

Comment:
Wonder if we should just define the defaults along side the types. Then we could share them like we do the types

Reply:
we can wait with that for now, I do not expect defaultData to be touch from outside